--- /tmp/std-orig.clj	2026-02-23 16:03:58.426443667 -0800
+++ /tmp/std-bb.clj	2026-02-23 16:03:58.431202619 -0800
@@ -475,14 +475,15 @@
   `(doall (for ~bindings (do ~@body))))
 
 (defn err [x]
-  (fmt.Fprint os.Stderr x))
+  (binding [*out* *err*]
+    (clojure.core/print x)))
 
 #_(defn eval [S]
   (ys/eval (str "!ys-0\n" S)))
 
 (defn exit
   ([] (exit 0))
-  ([I] (os.Exit I)))
+  ([I] (System/exit I)))
 
 ;; `if` is a special form in Clojure, but we can make resolve with this for use
 ;; in dot chaining.
@@ -542,21 +543,24 @@
 (intern 'ys.std 'write (fn [& args] (apply spit args)))   ; XXX
 
 (defn out [x]
-  (fmt.Fprint os.Stdout x))
+  (clojure.core/print x)
+  (flush))
 
 #_(defn pp [x]
   (pp/pprint x))
 
 (defn print [x]
-  (fmt.Fprint os.Stdout x))
+  (clojure.core/print x)
+  (flush))
 
 #_(def _println (resolve 'println))
 
 (defn say [x]
-  (fmt.Fprintln os.Stdout (str x)))
+  (println x))
 
 (defn warn [x]
-  (fmt.Fprintln os.Stderr (str x)))
+  (binding [*out* *err*]
+    (println x)))
 
 
 ;;------------------------------------------------------------------------------
